e2e/cypress/integration/notifications/browser_tab_notification_spec.js

```diff
@@ -0,0 +1,140 @@
+
+const timeouts = require("../../fixtures/timeouts");
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @notifications
+
+describe('Notifications', () => {
+    let user1;
+    let user2;
+    let team1;
+    let team2;
+    let testTeam1TownSquareUrl;
+    let testTeam2TownSquareUrl;
+
+    const timestamp = Date.now();
+
+    before(() => {
+        
+        cy.apiInitSetup().then(({team, user}) => {
+            user1 = user;
+            team1 = team;
+            testTeam1TownSquareUrl = `/${team.name}/channels/town-square`;
+
+            cy.apiCreateUser().then(({user: otherUser}) => {
+                user2 = otherUser;
+                cy.apiAddUserToTeam(team.id, user2.id);
+            });
+
+            cy.apiCreateTeam(`test-team-${timestamp}`, `test-team-${timestamp}`).then(({team: anotherTeam}) => {
+                team2 = anotherTeam;
+                testTeam2TownSquareUrl = `/${team2.name}/channels/town-square`;
+                cy.apiAddUserToTeam(team2.id, user1.id);
+                cy.apiAddUserToTeam(team2.id, user2.id);
+            });
+        });
+    });
+
+    it('MM-T560_1 Browser tab and team sidebar unreads and mentions - Mention in different team', () => {
+        
+        let originalFavicon = "";
+
+        // # User 1 view team A
+        cy.apiLogin(user1);
+        cy.visit(testTeam1TownSquareUrl);
+
+        // # Remove all unreads before test
+        cy.get("#publicChannelList").should("be.visible").get("li").eq(1).click();
+
+        // * Check for no unreads or mentions
+        cy.get(".unread-title").should("not.exist");
+
+        // # Save path of current favicon path
+        cy.get("link[rel=icon]").should("have.attr","href").then(($path)=> {
+            originalFavicon = $path;
+        });
+
+        // # Have another user view team B
+        cy.apiLogin(user2);
+        cy.visit(testTeam2TownSquareUrl);
+
+        // # Post a message with mentioning User 1 from Town Square
+        cy.postMessage(`@${user1.username}`);
+
+        // # Return to User 1
+        cy.visit(testTeam1TownSquareUrl);
+
+        cy.apiLogin(user1);
+        cy.visit(testTeam1TownSquareUrl);
+
+        // * Browser tab should displays (1) * channel - [team name] Mattermost
+        cy.get("title")
+            .contains("(1)");
+        // * Team sidebar: Small dot left of team B icon in team sidebar
+        cy.get(`#${team2.name}TeamButton`).parent(".unread").should("be.visible")
+            .within(()=>{ 
+        // * Team sidebar: a mention badge in top right corner of the badge with number "1"
+            cy.get(".badge").contains("1")
+            // # Make mention indicated
+            .click();
```

Comment:
I'm not sure if mention should be clicked. If I don't click this, the test will be fail (favicon turns red)